protostar/cairo/protostar_rust_bindings.py

```diff
@@ -0,0 +1,33 @@
+from typing import Optional
+from contextlib import contextmanager
+
+import rust_test_runner_bindings
+
+
+class CairoBindingException(Exception):
+    def __init__(self, message: str, details: Optional[str] = None):
+        self.message = message
+        self.details = details
+        super().__init__(message)
```

Comment:
Didn't we already have a class similar to this somewhere?

Reply:
We have the same class in compiler bindings

Reply:
correct, should be reused

Reply:
[make the structure of files more clear, reuse the exception code](https://github.com/software-mansion/protostar/pull/1881/commits/7f81331f897517135e4deefb9cbfdaa2c711f80d)